frontend/src/components/Header.tsx

```diff
@@ -49,9 +55,7 @@ const Header = ({ logoPath }: Props) => {
       <div
         className={`usa-overlay ${isMobileNavExpanded ? "is-visible" : ""}`}
       />
-      <GovBanner
-        language={i18n.language?.match(/^es-?/) ? "spanish" : "english"}
-      />
+      <GovBanner language={"english"} />
```

Comment:
Can we put `TODO` notes in places like this we know we'll need to revisit? Most strings will be easy to find. But this could go overlooked.

Reply:
There is a `// TODO: Remove during move to app router and next-intl upgrade` in the beginning of each implementation. I added one here as well.

```suggestion
      // TODO: Remove during move to app router and next-intl upgrade
      <GovBanner language={"english"} />
```